sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/nullExpressions.scala

```diff
@@ -357,7 +358,8 @@ case class AtLeastNNonNulls(n: Int, children: Seq[Expression]) extends Predicate
 
   override def doGenCode(ctx: CodegenContext, ev: ExprCode): ExprCode = {
     val nonnull = ctx.freshName("nonnull")
-    val code = children.map { e =>
+    ctx.addMutableState("int", nonnull, s"")
```

Comment:
s"" -> "".

Reply:
thanks! Anyway, I am refactoring this, since I figured out a way to avoid the declaration of a global attribute. I can't do the same for coalesce unfortunately, because there I'd need to return two values from the methods.